### ISSUE TITLE
Using formControlName, getting template parse error (says ngModel is not a known property of ng-select, but works fine without formControlName)

### ISSUE THREAD

**Issue by kraenels** [ng-select/ng-select#1226]
I'm auditing our app for WCAG and was happy to see "labelForId" and "formControlName" were already built in. The "labelForId" property works just fine, but "formControlName" throws a template parse error:
```
Error: Template parse errors:
Can't bind to 'ngModel' since it isn't a known property of 'ng-select'.
```

Here's the code:
```
<ng-select [items]="decisionPurposesBs | async" appendTo="body"
			bindLabel="label" bindValue="value"
			[(ngModel)]="entityDecision.decisionPurposeId"
			labelForId="purpose" formControlName="purpose">
</ng-select>
```

What is it about adding the "formControlName" property that's causing the error? If I remove "formControlName", there are no errors.

**Comment by arun-maddheshia** [comment]
@kraenels please check did you import FormsModule package in your App or Independent module.

**Comment by kraenels** [comment]
> @kraenels please check did you import FormsModule package in your App or Independent module.

Yes, FormsModule is imported.

I have "entity.module.ts" (independent module) importing "ControlsModule" (our custom UI modules):
`import { ControlsModule } from '@controls/controls.module';`
and
`imports: [ ControlsModule ]` (I have other imports, too, but didn't figure it was necessary to copy the whole file over. 

ControlsModule imports FormsModule:
`import { FormsModule } from '@angular/forms';`
and
`imports: [ FormsModule ]` (again, this is one of several other imports)

**Comment by arun-maddheshia** [comment]
@kraenels it is duplicate of https://github.com/ng-select/ng-select/issues/187

Its good if you provide a [stackblitz](https://stackblitz.com/edit/ng-select)

**Comment by varnastadeus** [comment]
FormControlName is part of reactiveFormsModule not FormsModule

**Comment by kraenels** [comment]
@arun-maddheshia If you read my original post, it's not an issue with [(ngModel)] in itself; it works just fine without formControlName. 

@varnastadeus I will try importing reactiveFormsModule, and if that still doesn't work, I will create a stackblitz. Please hold!

**Comment by kraenels** [comment]
@arun-maddheshia @varnastadeus I got it to work by importing CommonModule and FormsModule, ReactiveFormsModule in the independent module, EntityModule (which imports my template, entity-master). 

EntityModule had already imported other independent modules we call ControlsModule and SharedModule. ControlsModule imports CommonModule, FormsModule, and ReactiveFormsModule. The app.module imports BrowserModule.

If I'm using the ng-select in the entity-master template, looks like I have to import CommonModule and FormsModule & ReactiveFormsModule in the EntityModule which is in the same folder as entity-master... even though EntityModule already imports them by importing other modules that import them. Does the import of the FormsModule & ReactiveFormsModule _have_ to happen in the EntityModule in this case?

**Comment by varnastadeus** [comment]
use StackOverflow for this type of questions or read https://angular.io/guide/sharing-ngmodules
